package.json

```diff
@@ -20,7 +20,7 @@
     "es2015"
   ],
   "engines": {
-    "node": ">=6"
+    "node": ">=10"
```

Comment:
This change also affects users, because some package managers (such as yarn) will refuse to install it on Node.js 8.

Reply:
Thanks, I forgot Yarn actually checks this. 😅 I'll revert this change.